packages/angular/README.md

```diff
@@ -0,0 +1,237 @@
+<p align="center">
+  <a href="https://sentry.io" target="_blank" align="center">
+    <img src="https://sentry-brand.storage.googleapis.com/sentry-logo-black.png" width="280">
+  </a>
+  <br />
+</p>
+
+# Official Sentry SDK for Angular
```

Comment:
Can we mention the versions that it is compatible for and the relationship with the previous angular integration?

Reply:
I'll include this in the docs themselves when I'll be writing them.